app/src/main/java/com/github/multimatum_team/multimatum/activity/DeadlineDetailsActivity.kt

```diff
@@ -233,6 +233,17 @@ class DeadlineDetailsActivity : AppCompatActivity() {
         editMode = editMode.not()
     }
 
+    fun displayLocationOnMap(view: View) {
+        val deadline = deadlineListViewModel.getDeadline(id)
+
+        if (deadline.location != null) {
+            val intent = Intent(this, DisplayLocationActivity::class.java)
+            intent.putExtra("latitude", deadline.location.latitude)
+            intent.putExtra("longitude", deadline.location.longitude)
+            startActivity(intent)
+        }
```

Comment:
Really minor, and check the syntax if you apply it, but this seems a little better to me:
```suggestion
        deadline.location?.apply {
            val intent = Intent(this, DisplayLocationActivity::class.java)
            intent.putExtra("latitude", latitude)
            intent.putExtra("longitude", longitude)
            startActivity(intent)
        }
```

Reply:
Good suggestion